default.nix

```diff
@@ -1,46 +1,5 @@
-{ pkgs ? import <nixpkgs> {} }:
+with import <nixpkgs> {};
```

Comment:
I thought it was always a good idea to let the caller pass his own pkgs set, in case he want to override something. Isn't it?

Reply:
Indeed !